libs/ansible/src/const.ts

```diff
@@ -0,0 +1,20 @@
+import { ROUTE } from '@flightctl/ui-components/src/hooks/useNavigate';
+
+export const appRoutes = {
+  [ROUTE.ROOT]: '/edge/overview',
+  [ROUTE.FLEETS]: '/edge/fleets',
+  [ROUTE.FLEET_DETAILS]: '/edge/fleets',
+  [ROUTE.FLEET_CREATE]: '/edge/fleets/create',
+  [ROUTE.FLEET_EDIT]: '/edge/fleets/edit',
+  [ROUTE.FLEET_IMPORT]: '/edge/fleets/import',
+  [ROUTE.DEVICES]: '/edge/devices',
+  [ROUTE.DEVICE_DETAILS]: '/edge/devices',
+  [ROUTE.DEVICE_EDIT]: '/edge/devices/edit',
+  [ROUTE.REPO_CREATE]: '/edge/repositories/create',
+  [ROUTE.REPO_EDIT]: '/edge/repositories/edit',
+  [ROUTE.REPO_DETAILS]: '/edge/repositories',
+  [ROUTE.REPOSITORIES]: '/edge/repositories',
+  [ROUTE.RESOURCE_SYNC_DETAILS]: '/edge/resourcesyncs',
```

Comment:
I just realised that in `useNavigate` the Routes for Repository details, Fleet details and RS details contain an extra space.

Could you change them in this PR, please?

Reply:
Good catch, fixed